### PR TITLE
Copy transcoding code

### PR DESCRIPTION
**Pull request by qiwzhang**

### CODE REVIEW

File: contrib/endpoints/src/grpc/transcoding/bookstore.proto [hunk 1]
```diff
@@ -0,0 +1,77 @@
+// Copyright (C) Extensible Service Proxy Authors
```
Comment (kyessenov) [inline contrib/endpoints/src/grpc/transcoding/bookstore.proto]: Change license header.

File: contrib/endpoints/src/grpc/transcoding/BUILD [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#
+package(default_visibility = ["//visibility:private"])
```
Comment (kyessenov) [inline contrib/endpoints/src/grpc/transcoding/BUILD]: Make it public or same as api_manager?

**Comment by lizan** [comment]
You should keep BULD file, and change `protobuf_bzl` to `protobuf_git`, it should work.

**Comment by qiwzhang** [comment]
Not easy to make it bazel buildable. Need to change many includes too. We can do that later.  For now, not to add BUILD as not to break build for other codes.

**Pull request merged**